functions/venue.js

```diff
@@ -767,6 +768,135 @@ exports.updateVenue_v2 = functions.https.onCall(async (data, context) => {
   admin.firestore().collection("venues").doc(venueId).update(updated);
 });
 
+exports.updateVenueNG = functions.https.onCall(async (data, context) => {
```

Comment:
Why do we need a _third_ variant of updating venues?😃

Reply:
The other ones are creating venue objects with incorrect data that was used for admin v2, this one is going to create an empty object add only the passed fields and then update the firestore document